webapp/src/pages/Start.js

```diff
@@ -27,7 +27,10 @@ const Start = () => {
 
   const handleSubmit = (event) => {
     event.preventDefault();
-    setError();
+    validated.turnOn();
+    const phoneInput = document.querySelector("input");
```

Comment:
We should be using refs for this. But also I am not sure it is what we want.

```js
const phoneRef = React.useRef();

phoneRef.current.focus()

<Input ref=phoneRef ... />
```

Reply:
I don't think there is ever a case we should be using `document.querySelector` in React though it may be needed in some exotic integration cases.